docs/code.md

```diff
@@ -26,48 +26,40 @@ If you are an experienced open source contributor you may be familiar with this
 workflow. If you are new or just need reminders, the steps below link to more
 detailed documentation in Docker's project contributors guide.
 
-1. <a href="http://docs.docker.com/project/software-required/"
-target="_blank">Get the software</a> you need.
+1. [Get the software](project/software-required.md) you need.
 
 	This explains how to install a couple of tools used in our development
 	environment.  What you need (or don't need) might surprise you.
 
-2. <a href="http://docs.docker.com/project/set-up-git/"
-target="_blank">Configure Git and fork the repo</a>.
+2. [Configure Git and fork the repo](project/set-up-git.md).
 
 	Your Git configuration can make it easier for you to contribute.
 	Configuration is especially key if are new to contributing or to Docker.
 
-3. <a href="http://docs.docker.com/project/set-up-dev-env/"
-target="_blank">Learn to work with the Docker development container</a>.
+3. [Learn to work with the Docker development container](project/set-up-dev-env.md).
 
 	Docker developers run `docker` in `docker`.  If you are a geek,
 	this is a pretty cool experience.
-4. <a href="http://docs.docker.com/project/find-an-issue/"
-target="_blank">Claim an issue</a> to work on.
 
-	We created a filter listing <a href="http://goo.gl/Hsp2mk" target="_blank">all open
-	and unclaimed issues</a> for Docker.
+4. [Claim an issue](workflow/find-an-issue.md) to work on.
 
-5. <a
-href="http://docs.docker.com/project/work-issue/" target="_blank">Work on the
-issue</a>.
+	We created a filter listing 
+	[all open and unclaimed issues](https://goo.gl/Hsp2mk) for Docker.
```

Comment:
Same here with the short URL (but also needs confirmation of @moxiegirl in case they were deliberately added)